style.css

```diff
@@ -443,11 +475,13 @@ textarea::placeholder {
     margin-left: 100px;
     margin-right: 100px;
     border-radius: 24px;
+    padding-left: 8px;
+    padding-right: 8px;
   }
 
   .ruby {
-    padding: 4px 20px;
-    margin-right: -1px;
+    /* padding: 4px 20px; */
```

Comment:
Nice job so far👍. I suggest you take away every comment block of code in your project as it makes your code base look messy and less professional hence it is not a good practice👍.